src/app/donation-start/donation-start.component.ts

```diff
@@ -345,6 +348,7 @@ export class DonationStartComponent implements OnInit {
       'Remember every penny helps & you can continue to make an unmatched donation to the charity!',
       'Cancel',
       donation,
+      this.surplusDonationInfo,
```

Comment:
@tbg-jasonfung  I still don't understand why we're passing this around, or adding an extra class var? We already have `campaign` as a class var, let's just use that and avoid extraneous inter-method params.